os/kernel/Kconfig

```diff
@@ -999,6 +999,7 @@ endmenu # Stack size information
 
 config LOG_DUMP
 	bool "Enable log dump functionality"
+	depends on LIB_LZMA || LIB_MINIZ
```

Comment:
Does it use lzma and miniz directly? I think this module uses compression function and compression function use miniz or lzma. If this is right, then, need to be compression config for here.

Reply:
yes, the miniz and lzma library functions are used directly. There is no dependency with compression module

Reply:
Then, there is no reason why we keep the compression module.